cypress.config.ts

```diff
@@ -13,7 +21,192 @@ export default defineConfig({
     POSTGRES_URL: 'postgresql://postgres:postgres@localhost:5432/registry',
   },
   e2e: {
-    setupNodeEvents(on) {
+    setupNodeEvents(on, config) {
+      async function connectDB(query: string) {
+        const dbUrl = new URL(config.env.POSTGRES_URL);
+        const client = new pg.Client({
+          user: dbUrl.username,
+          password: dbUrl.password,
+          host: dbUrl.hostname,
+          database: dbUrl.pathname.slice(1),
+          port: Number(dbUrl.port),
+          ssl: false,
+        });
+        await client.connect();
+        const res = await client.query(query);
+        await client.end();
+        return res.rows;
+      }
+
+      on('task', {
+        connectDB,
+        async deleteUser(email = 'test@test.com') {
```

Comment:
In integration tests, we create new users with a new org for each test case. Unless there is a clear reason for doing the clean up we can drop it, as it adds additional stuff to maintain.

Reply:
yeah would be nice to have 1 proper seeding

Reply:
I create separated PR for this refactoring, but not sure why can't start docker containers locally
https://github.com/graphql-hive/platform/pull/5881
```
ERROR: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: unsupported frontend capability moby.buildkit.frontend.contexts
/Users/dmytro/Desktop/GUILD/graphql-hive/integration-tests:
```